### ISSUE TITLE
Deprecate `Tooltip` and/or convert to `system_arguments`

### ISSUE THREAD

**Issue by khiga8** [primer/view_components#663]
Here are some things that came up during pairing session with @manuelpuyol 

- Should `Tooltip` even be a component?
- `Tooltip` cannot be added to a slot component. 
- We actively discourage people from using `Tooltip` via documentation and linter so it's odd to even have it as part of the system.

Feel free to comment below if you can think of reasons to keep the `Tooltip` component!
